<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>849</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Language Cards</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>881</width>
      <height>841</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <attribute name="title">
      <string>My Cards</string>
     </attribute>
     <widget class="QPushButton" name="addButton">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>821</width>
        <height>791</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>819</width>
         <height>789</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="cardsQuantity">
      <property name="geometry">
       <rect>
        <x>833</x>
        <y>190</y>
        <width>36</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>0 Cards</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="clearButton">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>50</y>
        <width>31</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>C
 l
 e
 a
 r</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Tests</string>
     </attribute>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>851</width>
        <height>791</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QPushButton" name="startButton">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>460</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Start Test</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineQuantity">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>320</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineHours">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>320</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>1</number>
        </property>
        <property name="placeholderText">
         <string>00</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>280</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Enter time:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineMinutes">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>320</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>2</number>
        </property>
        <property name="placeholderText">
         <string>00</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineSecunds">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>320</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>2</number>
        </property>
        <property name="placeholderText">
         <string>00</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>300</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>m:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>300</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>s:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>300</y>
          <width>21</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>h:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>280</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Enter cards quantity:</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="ErrorLabel2">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>380</y>
          <width>221</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>OCR A Extended</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="timeLabel">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>360</y>
          <width>141</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>00:00:00</string>
        </property>
       </widget>
       <widget class="QLabel" name="dateLabel">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>20</y>
          <width>221</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Current date:</string>
        </property>
       </widget>
       <widget class="QLabel" name="cardsQuantity2">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>320</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QLabel" name="timerLabel">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>20</y>
          <width>151</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>00:00:00</string>
        </property>
       </widget>
       <widget class="QPushButton" name="confirmAnswer">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>480</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Confirm</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>390</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Answer: </string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineForAnswer">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>410</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>20</number>
        </property>
        <property name="placeholderText">
         <string>Enter Text</string>
        </property>
       </widget>
       <widget class="QLabel" name="textToTranslate">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>310</y>
          <width>391</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Translate: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="cardsLeft">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>140</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Cards left:</string>
        </property>
       </widget>
       <widget class="QLabel" name="cardsGuessed">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>160</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Cards guessed: 0</string>
        </property>
       </widget>
       <widget class="QLabel" name="cardsNGuessed">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>190</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Cards not guessed: 0</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelError3">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>610</y>
          <width>251</width>
          <height>81</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="correctLabel">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>510</y>
          <width>251</width>
          <height>81</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="closeTestButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>710</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>450</y>
          <width>171</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>9</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Letter case matters</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Statistics</string>
     </attribute>
     <widget class="QListWidget" name="statistics">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>281</width>
        <height>751</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="learningStats">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>93</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Learning</string>
      </property>
     </widget>
     <widget class="QPushButton" name="appStats">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>93</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>App Using</string>
      </property>
     </widget>
     <widget class="QStackedWidget" name="informationStats">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>70</y>
        <width>551</width>
        <height>731</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_3">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="statisticsLabel">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>678</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_4">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <family>Rockwell</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Choose statistics</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
